dags/deletes.py

```diff
@@ -447,11 +577,12 @@ def count_pending_deletes(client: Client) -> int:
 
     delete_mutation_runner = LightweightDeleteMutationRunner(
         EVENTS_DATA_TABLE(),
-        "(dictHas(%(dictionary)s, (team_id, %(person_deletion_type)s, person_id)) AND timestamp <= dictGet(%(dictionary)s, 'created_at', (team_id, %(person_deletion_type)s, person_id))) OR (dictHas(%(dictionary)s, (team_id, %(team_deletion_type)s, team_id)))",
+        "(dictHas(%(pending_deletes_dictionary)s, (team_id, %(person_deletion_type)s, person_id)) AND timestamp <= dictGet(%(pending_deletes_dictionary)s, 'created_at', (team_id, %(person_deletion_type)s, person_id))) OR (dictHas(%(pending_deletes_dictionary)s, (team_id, %(team_deletion_type)s, team_id))) OR (dictHas(%(adhoc_event_deletes_dictionary)s, (team_id, uuid)))",
```

Comment:
This line is getting very long and hard to read - could be good to split it up into multiple lines and use the `or(…, …, …)` function instead of `OR` operator just to help with readability